static/style.css

```diff
@@ -124,11 +124,11 @@
 }
 
 input:checked + .slider {
-  background-color: #2196F3;
+  background-color: #205572;
```

Comment:
This is the direction the issue was meant. Maybe in the `index.html` the font color, ... could be adapted to the colors instead of plain black.